velox/functions/remote/CMakeLists.txt

```diff
@@ -15,3 +15,5 @@
 add_subdirectory(if)
 add_subdirectory(client)
 add_subdirectory(server)
+add_subdirectory(utils)
+add_subdirectory(benchmarks)
```

Comment:
Benchmarks must be enclosed under the `VELOX_ENABLE_BENCHMARKS` CMake option.
CC: @amitkdutta
@czentgr is opening a fix.